app/models/user.rb

```diff
@@ -26,12 +26,12 @@ def induction_coordinator?
     induction_coordinator_profile.present?
   end
 
-  def early_career_teacher?
-    early_career_teacher_profile.present?
+  def lead_provider_admin?
+    lead_provider_profile.present?
```

Comment:
I don't quite get this: so every lead provider is a lead_provider_admin? Why would we have both if that is the case?

Reply:
Every lead provider user is a lead provider admin. For time being anyway - we can change what this function does when we have more than one type of lead provider user.

Reply:
I suppose my question was why we didn't just call this lead_provider? especially when lead_provider and lead_provider_admin? are the same. lead_provider_admin? introduces the concept of an admin that we don't have nor seem to need right now.

Reply:
Cause `lead_provider?` sounds weird in my head as a method describing a type of the user. And it is very similar to an existing method, `lead_provider` (only `?` of difference) - meaning it's easier to make a mistake between the two.

I think the only user idea that we have around lead providers is the person looking for schools, and eventually filling in a bunch of forms around partnerships - seems admin enough to me. Happy to change admin to something more descriptive, just don't know what.

Reply:
I'm with Alfred on this one

Reply:
Both points are valid and you don't need to change it if that is the case. I just needed the explanation / clarification.

Reply:
Fine, but I reserve the right to "I told you so" when someone forgets the question-mark 😆

Reply:
How un rails-y would `is_lead_provider?` be?

Reply:
As un-railsy as having a rubocop against it but most rails codebases tend to have some of these.